### ISSUE TITLE
Toast: outline stats on screen after toast dismissed

### ISSUE THREAD

**Issue by hipstersmoothie** [radix-ui/primitives#1212]
## Bug report

Happens on iPhone 

### Current Behavior

![image](https://user-images.githubusercontent.com/1192452/156107963-e2b046bc-f8b9-48f0-a68d-d6f58de1ddd2.png)

**Comment by jjenzz** [comment]
Oh snap, I raised this with the team last night. @benoitgrelard looks like this issue needs to be moved to the website repo.

**Comment by tmilewski** [comment]
@jjenzz I'm seeing this on our Toast implementation as well, so I'm not sure that it should live on the Radix site repo. Unless I'm doing something wrong... which is entirely possible ;-).

**Comment by jjenzz** [comment]
@tmilewski since the radix primitives are packaged without styles the intention is that consumers would handle styling, however, this one is a bit of a grey area. I'm not sure whether this is something primitives should handle or not but I'm not on the team anymore so perhaps @benoitgrelard or @andy-hook can help.

For now, you should be able to add `outline: none` to the `Toast.Viewport`.

**Comment by benoitgrelard** [comment]
I agree it should be dealt with at the app level I think, so I'll move the issue to the website repo for now.
